doc/conf.py

```diff
@@ -163,7 +163,8 @@
 html_theme = "sphinx_rtd_theme"
 html_theme_options = {
     "logo_only": True,
-    "prev_next_buttons_location": None
+    "prev_next_buttons_location": None,
+    "navigation_depth": 5,
```

Comment:
In the past navigation depth used to increase build time significantly, just check before merging that things are ok

Reply:
Ah, good to know. I couldn't see any difference between depth=3 and depth=5 so I guess this isn't an issue anymore :)